packages/devtools/lib/src/memory/memory.dart

```diff
@@ -144,6 +206,22 @@ class MemoryScreen extends Screen {
     }
   }
 
+  // TODO(terry): Need to record the GC event in the timeline.
```

Comment:
do we have to record it? I would hope we would still get a GC event back even if we manually triggered the GC. That way the wacky case of multiple memory profiler users would still work.

Reply:
No, the GC itself will cause the event to fire.